exercises/wordy/example.py

```diff
@@ -13,7 +13,7 @@
 
 def answer(question):
     if not (question.startswith("What is ") and question.endswith("?")):
-        raise ValueError("Ill-formed question")
+        raise ValueError("unknown operation")
```

Comment:
Please revert this. The correct message was already being used.
```suggestion
        raise ValueError("Ill-formed question")
```